### ISSUE TITLE
Staging Review [Print Styles Design]

### ISSUE THREAD

**Issue by jonathanjnelson** [department-of-veterans-affairs/va.gov-team#6364]
## Steps to complete Staging Review: 

- [x ] Product Manager: create this issue and fill in feature name in the title and other bolded information appropriately
- [x ] Link to this issue once created in #vfs-platform-support in Slack; tag @ Megan Kelley @ AndreaHewitt
- [ ] Andrea and Megan to schedule meeting with VSP reviewers and **requesting team** attendees (as listed below)
- [ ] At least 2 days before scheduled meeting, you must complete list of artifacts below
- [ ] VSP <> **requesting team** review meeting completed
- [ ] Practice area reviewers create sub-issues attached to this one with feedback for your team on *feature name*
- [ ] **Requesting team** completes attached feedback tickets based on definition of done included in each practice area feedback ticket (may differ for each practice area, please refer to attached tickets for further instruction)
- [ ] All requesting team attendees complete brief [VSP Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey

## Artifacts - _please provide the following artifacts at least 2 days before scheduled review meeting_
- Link to product outline https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/print-styles/outline.md

- Print styles: https://vsateams.invisionapp.com/share/ATW7AI7QXCV

- Link to [release plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/release-plan-template.md) with the "Planning" sections completed (in each section: Phase I, Phase II, Go Live) **N/A**
- Access information for the tool or feature in staging _**do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository**_  **N/A**
- Link to your product test cases/test plan in TestRail with test execution logs ([learn more](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)) _for QA practice area review_ **N/A**
- Link to your [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/12) and [Summary(Defects)](https://dsvavsp.testrail.io/index.php?/reports/view/14) reports in TestRail _for QA practice area review_  **N/A**
- Link to your accessibility test plan: **N/A**
  - **TestRail** [TestRail VSP accessibility test plan template] (https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) **OR**
  - **Markdown** [Github accessibility test plan template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-plan-template.md)
- List of known issues (with links to tickets when applicable) **N/A**

## Meeting attendees from **requesting team**
- Product Manager (required): N/A
- DEPO product lead (required): N/A
- Anyone else from your team who significantly contributed to the artifacts provided (required): **Riley Orr**,  **Jennifer Strickland** (optional) 

## Scheduling
Preferred dates or times: 

3/5/20 - 4 PM - 5 PM EST 
3/6/20 - 3 PM - 5 PM EST 
3/9/20 - 2 PM - 3 PM EST, 4:30 - 5 PM EST 

## Additional Notes
During this collaboration point, you will receive feedback from the following practice areas: design, accessibility, content, QA, and optional: product, analytics, engineering, information architecture

**Comment by meganhkelley** [comment]
Hey @jonathanjnelson ! Can you please give a bit more context on what feedback you are looking for here, and what stage of the development cycle you are in? From the info you've provided, it looks like you are looking for feedback on the print styles as templates only — is that correct? Are there plans (by your team or another) to implement these on VA.gov? 

With the assumption that you're just looking for feedback on the _templates_, I'll go ahead and get this scheduled with the following practice areas: design, accessibility, content, (and information architecture as optional). Because (I think) this does not detail actual implementation of these print styles, I will leave out: analytics, engineering, QA, and product. Future implementation would efforts would need to check back in as part of the VSP Collaboration Cycle for further review. Does that line up with what you're looking for and your expectations for this collaboration point? 

Lastly, I'm having trouble getting a meeting in any of your suggested times with the full group. Would you be able to do Thursday 3/5 between 1:30 - 2:30 pm ET, or Tuesday 3/10 between 11:30 - 12 pm ET?

**Comment by jonathanjnelson** [comment]
Of course, I do apologize for being vague (I’ve never utilized a template like this before). 

Yes, you are correct. Riley Orr and I are looking for feedback on print styles templates only. This is a side project we are working on, and the goal is to design the print stylesheet and standardize across all VA.gov pages (and eventually implement). With that being said, I feel that review with design, accessibility, content, (and information architecture as optional) makes sense at this point in the project. 

It looks like Thursday 3/5 between 1:30 - 2:30 pm ET will work!

**Comment by meganhkelley** [comment]
Excellent! Scheduled for 3/5 @ 2 pm ET. Is there a DEPO product owner on yours or Riley's team who should be included?

**Comment by jonathanjnelson** [comment]
Thank you so much! 

I would like to mention that we have met with Jennifer and Trevor multiple times. Just so I am clear, we would have two checkpoints? 

Also, we don't have a product owner but I am going to check on that today.

**Comment by meganhkelley** [comment]
Oh, good to know @jonathanjnelson ! 

@1Copenut @jenstrickland — because y'all have already seen these print styles a fair bit it sounds like, your feedback for the 3/5 review can be optional. 

Thanks all! LMK if you have any further questions. 👍

**Comment by 1Copenut** [comment]
VSP accessibility comment for the print styles. Just one **✔️ Consider** in the attached screenshot below. Please holler at me if you want to discuss further or have any questions.

---
![Screen Shot 2020-03-10 at 11 22 32 AM](https://user-images.githubusercontent.com/934879/76335440-87d0b180-62c2-11ea-9cbc-92f828a30d3f.png)

**Comment by jonathanjnelson** [comment]
Thank you for all the feedback on the designs. Riley made revisions and we’ve decided to move forward with the following designs and strive to implement in the Caregiver’s MVP. 

Please let us know if you have any questions and concerns. 

Thank you, 

Riley and Jonathan 


![Caregiver confirmation - Desktop - v 02](https://user-images.githubusercontent.com/58444931/77105597-e812d200-69eb-11ea-983b-5a3026a530ff.png)
![Order confirmation - Desktop - v 02](https://user-images.githubusercontent.com/58444931/77105614-ecd78600-69eb-11ea-84c1-f3dbb55358b0.png)

**Comment by meganhkelley** [comment]
Hi @jonathanjnelson ! Was this ever implemented? We may have found a bug with the VA.gov logo appearing on this page in prod — https://dsva.slack.com/archives/CQH357ZTP/p1602626937141100

I'm not positive it's related to the work on print styles, but I thought I recognized the design from this work y'all had done. :) 

It's a trivial bug, just looking for info! 

cc @billfienberg

**Comment by jenstrickland** [comment]
It looks like it is a broken url — which is why the image isn't loading.

**Comment by jonathanjnelson** [comment]
@meganhkelley It was implemented for the 10-10CG. Let me know what I can do to assist.
